contribute.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Phabricator",
+  "description": "Mozilla's Phabricator instance for code review.",
+  "repository": {
+    "url": "https://github.com/mozilla-services/mozphab",
+    "license": "MPL2",
```

Comment:
Hm. Phabricator itself is not MPL2, so I don't think this is right... and regardless there's no LICENSE file in there. I would check with an expert (like Gerv) on how we should license all the build stuff in that repo.

Reply:
I will reach out to gerv on irc :)